libraries-5/src/test/java/com/baeldung/githubapi/ClientUnitTest.java

```diff
@@ -0,0 +1,39 @@
+package com.baeldung.githubapi;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.GitHub;
+import org.kohsuke.github.GitHubBuilder;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ClientUnitTest {
+
+    @Test
+    void whenWeCreateAnAnonynousClient_thenWeCanAccessTheGithubApi() throws IOException {
+        GitHub gitHub = GitHub.connectAnonymously();
+
+        String apiUri = gitHub.getApiUrl();
+        assertEquals("https://api.github.com", apiUri);
+    }
+
+    @Test
+    @Disabled // Needs credentials configuring
```

Comment:
Can we specify how somewhere, maybe in a comment at the top of the file, so if a reader checks out the repo they understand what to do?